### PR TITLE
[DOC] Add docs and CHANGELOG record for disabling ownerReference on CA secrets

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

In #3453 we forgot to update the Docs and the CHANGELOG.md file. This PR fixes it.

### CODE REVIEW

**Review by ppatierno — APPROVED** [review]

File: documentation/modules/security/ref-certificates-and-secrets.adoc [hunk 1]
```diff
@@ -162,6 +162,29 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
 It needs to be accessible to the Strimzi components and for administrative access if you are intending to issue application certificates without using the User Operator.
 You can enforce this using Kubernetes role-based access controls if necessary.
 
+== Disabling `ownerReference` in the CA Secrets
+
+By default, the CA secrets created by Strimzi will have the `ownerReference` field set to the `Kafka` custom resource.
```
Comment (PaulRMellor) [inline documentation/modules/security/ref-certificates-and-secrets.adoc:167]:
```suggestion
By default, the CA secrets created by Strimzi have the `ownerReference` field set to the `Kafka` custom resource.
```

File: documentation/modules/security/ref-certificates-and-secrets.adoc [hunk 2]
```diff
@@ -162,6 +162,29 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
...
+This means that when the custom resource is deleted, the CA secrets will be garbage collected (deleted) by Kubernetes as well.
```
Comment (PaulRMellor) [inline documentation/modules/security/ref-certificates-and-secrets.adoc:168]:
```suggestion
This means that when the `Kafka` custom resource is deleted, the CA secrets are garbage collected (deleted) by Kubernetes as well.
```

File: documentation/modules/security/ref-certificates-and-secrets.adoc [hunk 3]
```diff
@@ -162,6 +162,29 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
...
+
+If you want to re-use the CA for a new cluster, you can disable the `ownerReference`.
+When disabled, the CA secrets will not be deleted by Kubernetes together with the corresponding `Kafka` custom resource.
```
Comment (PaulRMellor) [inline documentation/modules/security/ref-certificates-and-secrets.adoc:171]:
```suggestion
When disabled, CA secrets are not deleted by Kubernetes when the corresponding `Kafka` custom resource is deleted.
```

File: documentation/modules/security/ref-certificates-and-secrets.adoc [hunk 4]
```diff
@@ -162,6 +162,29 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
...
+
+.Example Kafka configuration with disabled `ownerReference` for Cluster and Client CAs
```
Comment (PaulRMellor) [inline documentation/modules/security/ref-certificates-and-secrets.adoc:173]:
```suggestion
.Example Kafka configuration with disabled `ownerReference` for cluster and client CAs
```

File: documentation/modules/security/ref-certificates-and-secrets.adoc [hunk 5]
```diff
@@ -162,6 +162,29 @@ NOTE: `_<cluster>_-clients-ca` is used to sign certificates of client applicatio
 It needs to be accessible to the Strimzi components and for administrative access if you are intending to issue application certificates without using the User Operator.
 You can enforce this using Kubernetes role-based access controls if necessary.
 
+== Disabling `ownerReference` in the CA Secrets
+
+By default, the CA secrets created by Strimzi will have the `ownerReference` field set to the `Kafka` custom resource.
+This means that when the custom resource is deleted, the CA secrets will be garbage collected (deleted) by Kubernetes as well.
+
+If you want to re-use the CA for a new cluster, you can disable the `ownerReference`.
```
Comment (PaulRMellor) [inline documentation/modules/security/ref-certificates-and-secrets.adoc:170]:
```suggestion
If you want to reuse the CA for a new cluster, you can disable the `ownerReference`.
```

**Review by PaulRMellor — APPROVED** [review]
Looks good. Few minor suggestions

**Pull request merged**